src/sentry/migrations/0171_backfill_cool.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 1.11.29 on 2021-03-11 06:29
+
+from django.db import migrations
+
+
+def backfill_is_cool(apps, schema_editor):
+    User = apps.get_model("sentry", "User")
+
+    for user in User.objects.all():
```

Comment:
The `User.objects.all()` call will load every user into memory which might cause problems due to thrashing or what have you. I'd suggest rewriting the function as:
```
User = apps.get_model("sentry", "User")

for user in RangeQuerySetWrapperWithProgressBar(User.objects.all()):
    is_cool = ...
    user.update(is_cool=is_cool)
```
An appealing solution might be to code this in raw SQL, but the niceness you get from `RangeQuerySetWrapperWithProgressBar` is better.